jdaviz/components/viewer_data_select.vue

```diff
@@ -107,6 +108,7 @@
         </div>
       </v-list>
     </v-menu>
+    <div ref="placeholder" :id="'target-' + viewer.id"></div>
```

Comment:
is this ref ever used (I don't see anywhere in the final diff) or just referred to by target?

Reply:
Good catch, and thank you for pointing that out. Initially, my intention was to implement this feature in a Vue-centric way, and the ref on the div was a part of that approach. I will remove the redundant reference in the upcoming commit.